### PR TITLE
process needs to be disposed of

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: src/Microsoft.Tye.Hosting/ProcessRunner.cs [hunk 1]
```diff
@@ -347,6 +347,12 @@ private void WriteReplicaToStore(string pid)
             return null;
         }
 
+        private static string GetEntryPointFilePath()
+        {
+            using var process = Process.GetCurrentProcess();
```
Comment (davidfowl) [inline src/Microsoft.Tye.Hosting/ProcessRunner.cs]: This doesn't seem right.

Comment (davidfowl) [reply src/Microsoft.Tye.Hosting/ProcessRunner.cs]: Does this return a new instance every time?

Comment (SimonCropp) [reply src/Microsoft.Tye.Hosting/ProcessRunner.cs]: yep

```
public static Process GetCurrentProcess()
{
	return new Process(".", isRemoteMachine: false, NativeMethods.GetCurrentProcessId(), null);
}
```

Comment (SimonCropp) [reply src/Microsoft.Tye.Hosting/ProcessRunner.cs]: i think "returns new instance" is meant to be implied by the fact that it is a method and not a property

**Pull request merged**